insights/src/main/resources/subsystem-templates/insights.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--  See src/resources/configuration/ReadMe.txt for how the configuration assembly works -->
+<config default-supplement="default">
+   <extension-module>org.jboss.as.ejb3</extension-module>
+   <subsystem xmlns="urn:jboss:domain:ejb3:3.1">
```

Comment:
this template is all wrong, this is ejb3 subsystem, you need to have your default config for your subsystem here, that gets shipped in standalone*/domain.xml files